RentItAll/src/items/ControllerServlet.java

```diff
@@ -0,0 +1,120 @@
+package items;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.List;
+
+import javax.servlet.GenericServlet;
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+public class ControllerServlet extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+    private ItemDBC itemDBC;
+    
+    public void init() {
+        String jdbcURL = getServletContext().getInitParameter("jdbcURL");
```

Comment:
Init is not needed. You can craete a default constructor of the servlet and call getConnection function
Connection con = itemDBC.getConnection();